test/extended/builds/docker-build-no-outputname.go

```diff
@@ -0,0 +1,39 @@
+package builds
+
+import (
+	"fmt"
+	"path/filepath"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("builds: Docker image buildconfig without output.to.Name specified", func() {
```

Comment:
maybe this deserves a better name... "builds: no name specified for the output image"